app/lib/database_connection.rb

```diff
@@ -0,0 +1,15 @@
+require 'pg'
+
+class DatabaseConnection 
+  def self.setup(db)
+    @connection = PG.connect(dbname: db)
```

Comment:
Could you not have made this method do the environment variable 'test' checks and pass on the appropriate connection to be used in your other classes like Peeps and User? Might have saved you the trouble of having to re-write the syntax for that in methods such as the 'self.all', 'self.create' etc methods. This way you can just connect to the appropriate database as a single line in such methods like databaseconnect = DatabaseConnection.setup then do whatever you need like databaseconnect.exec(----).